bedrock/firefox/templates/firefox/new/batm/private.html

```diff
@@ -2,18 +2,18 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% from "macros.html" import google_play_button with context %}
+{% add_lang_files "firefox/new/batm" %}
 
 {% extends "firefox/new/batm/scene1.html" %}
 
-{% block optimizely %}
-  {% if switch('firefox-new-batmprivate-optimizely', ['en-US']) %}
-    {% include 'includes/optimizely.html' %}
+{% block experiments %}
+  {% if switch('experiment-firefox-new-batm-anim', ['en-US']) %}
+    {% javascript 'experiment_firefox_new_batm_anim' %}
   {% endif %}
 {% endblock %}
 
 {% block main_header_copy %}
-<p id="main-header-copy" data-experience="batmprivate">
+<p id="main-header-copy" data-experience="batmprivate" data-variant="a">
```

Comment:
Just noticed - do we need this `data-variant="a"` attribute? I don't think it ever does anything, as there's no JS bundle in this template...